app/application/serializer.js

```diff
@@ -0,0 +1 @@
+export { default } from 'ember-local-storage/serializers/serializer';
```

Comment:
Same thing like in adapter:
```suggestion
import RESTSerializer from '@ember-data/serializer/rest';
import LocalStorageSerializer from 'ember-local-storage/serializers/serializer';
import ENV from 'mms-ember-project/config/environment';

const isTesting = ENV.environment === 'test';

const Serializer = isTesting ? RESTSerializer : LocalStorageSerializer;

export default class ApplicationSerializer extends Serializer {}
```